utils/install-dependencies.sh

```diff
@@ -26,7 +26,7 @@ function detect_aur_helper() {
         AUR_HELPER=pacaur
     else
         echo No available AUR helpers found. Please specify your AUR helper by AUR_HELPER.
-        exit -1
+        exit 1
```

Comment:
Why need to change the arch Linux installation?

Reply:
> Every command returns an exit status (sometimes referred to as a return status or exit code). A successful command returns a 0, while an unsuccessful one returns a **non-zero value** that usually can be interpreted as an error code. Well-behaved UNIX commands, programs, and utilities return a 0 exit code upon successful completion, though there are some exceptions.

Ref to [shell exit-status](https://tldp.org/LDP/abs/html/exit-status.html) and [SC2242](https://github.com/koalaman/shellcheck/wiki/SC2242)

IMO, We should obey `SC2242` to make it more user friendly ?

Reply:
Could we use `exit 255` to be compatible with the previous code?